src/mwocp68.ron

```diff
@@ -1,12 +1,17 @@
 (
     all: [],
     numerics: [
+        ("FAN_CONFIG_1_2", Raw, Unitless),
```

Comment:
For our current use case (debug dumping), it did not seem worth it to add this (or the other `Raw`, `Unitless` commands added in this PR) to `structured` and spell out all of their fields. If you disagree I can go back and fill these in.